torch/csrc/jit/passes/onnx/remove_inplace_ops_for_onnx.cpp

```diff
@@ -93,65 +93,52 @@ Value* MatchIfBlocksOutputForValue(
   return outer_block->owningNode()->outputs().at(output_size - 1);
 }
 
-// clang-format off
```

Comment:
nit: you can keep this line, that will help preserving the below aten graphs from being wrapped by clang-tidy.